domains/clayza.json

```diff
@@ -0,0 +1,9 @@
+{
+    "repo": "https://github.com/ClayzaAubert/clayzaaubert.github.io",
+    "owner": {
+        "username": "ClayzaAubert",
```

Comment:
You need to have either a email field, OWL field, or an alternative contact field.